server/routes/profile.js

```diff
@@ -16,7 +17,9 @@ router.route("/").get(protect, loadProfiles); //get all the profiles
 
 router.route("/").post(protect, createProfile); //create a new profile
 
-router.route("/:id").get(protect, getProfile); //Get a profile with ID
+router.route("/:id").get(protect, getProfile); //Get a profile with  profileID
+
+router.route("/user/:id").get(getProfileByUser); //Get profile with UserID
```

Comment:
Protect has been added and route is now moved to /routes/user.js  where it will return the profile of person who is logged in.
and the route is change to /users/profile